webapp/src/index.js

```diff
@@ -15,13 +15,22 @@ import Client from './client';
 class Plugin {
     // eslint-disable-next-line no-unused-vars
     initialize(registry, store) {
-        registry.registerChannelHeaderButtonAction(
-            <Icon/>,
-            (channel) => {
-                startMeeting(channel.id)(store.dispatch, store.getState);
-            },
-            'Start Webex Meeting'
-        );
+        const helpText = 'Start Webex Meeting';
+        const action = (channel) => {
+            startMeeting(channel.id)(store.dispatch, store.getState);
```

Comment:
Totally optional, but this should have been:
```suggestion
            store.dispatch(startMeeting(channel.id));
```